nx-X11/programs/Xserver/hw/nxagent/Render.h

```diff
@@ -19,6 +19,7 @@
 #define __Render_H__
 
 #include "screenint.h"
+#include "cursor.h"
```

Comment:
> Added because of undefined functions? (Just asking.)

In the original ``mitrap.c`` file the ``mi.h`` header file gets included (which includes ``cursor.h``). In NXmitrap.c, also Render.h is included (after ``mi.h`` got already included), using symbols from ``cursor.h``.

In the new version of NXmitrap.c, we do this:

```
#include "Render.h"

#include "../../render/mitrap.c" /* (where mi.h gets included) */

[...]
```

So the include order has changed and Render.h complains about undeclared symbols / types (or what it was).